README.md

```diff
@@ -4,15 +4,70 @@
 
-## Running locally
+## Table of contents
 
-Clone this repo with checkout to branch `master-2.0`. Run the `DiscordBot` project. :bangbang: Before running remember to add to the main direcotry file `token.txt` with discord bot token.
+1. [Running bot](#tc1)
+    1. [Important informations](#tc11)
+    2. [Local run](#tc12)
+    3. [Doker run](#tc13)
+        1. [Enviroment informations](#tc131)
+        2. [Using docker cli](#tc132)
+        3. [Using docker compose](#tc133)
+2. [Debug and development](#tc2)
+    1. [Database schemas](#tc21)
 
-## Running in docker
+## Running bot <a id="tc1"></a>
 
-### Using docker compose
+This part is about how to run bot depending on enviroment used.
 
-Container can be run using provided docker compose file provided example provided in `docker-compose-template.yaml`, or from the example below.
+### Imporatnt informations <a id="tc11"></a>
+
+Before trying to run you need to aquire token from [Discord](https://discord.com/developers/applications). 
+
+### Local run <a id="tc12"></a>
+
+Running locally is not advised as requires use of Visual Studio and installation of .Net 6. For debbuging it is prefered method.
+
+1. Clone this repo with checkout to branch `master-2.0`.
+2. Open `DiscordBot.sln` with Visual studio
+3. Place `token.txt` file in main folder with token aqured earlier.
+4. Run the `DiscordBot` project
+
+If there are some issues with packages try reimporting nu-get.
+
+### Docker run <a id="tc13"></a>
+
+Here steps for use of deployable version of DiscordBot. Default Docker Hub image could be found [here](https://hub.docker.com/r/seker212/discord_bot)
+
+#### Enviroment informations <a id="tc131"></a>
+
+Running in docker requires setting important evnviromental variables as well as some volumes for files that should be accessible after shutdown of container.
+
+Enviroment variables:
+
+| Name |Value|Purpose  |
+|----------|:-------------:|------|
+| TOKEN |  string (long chain) | Required. Discord bot token can't be run without it. |
+
+Volumes:
+
+| Name |      Container Path      |  Purpose |
+|----------|:-------------:|------|
+| Audio |  `/app/audio` | All sounds files are stored here |
+| Data |  `/app/data` | Database file is stored here with all servers configurations |
+| Logs |  `/app/logs` | Runtime logs are stored in this folder|
+
+#### Using docker cli  <a id="tc132"></a>
+
+Pulling and running container could be done using command below, simply place the TOKEN and name, and set the volume path in PWD.
+
+```bash
+docker run --rm -d -e "TOKEN=<discord app client secret here>" --name <name> -v <PWD>/audio:/app/audio seker212/discord_bot:2.0
+```
+
+#### Using docker compose <a id="tc133"></a>
+
+Use provided `docker-compose-template.yaml` or example below. Replace required values and run by using `docker compose up -d` command for Docker v2 or `docker-compose up -d` for v1.
```

Comment:
This won't work. The compose file either has to be renamed or passed to the command